scripts/ci-entrypoint.sh

```diff
@@ -212,6 +212,18 @@ if [[ -n "${TEST_CCM:-}" ]]; then
     fi
 
     export CCM_LOG_VERBOSITY="${CCM_LOG_VERBOSITY:-4}"
+    # Get CCM cluster CIDRs from cluster config template if not set
+    if [[ -z "${CCM_CLUSTER_CIDR:-}" ]]; then
+        curl -LO https://github.com/mikefarah/yq/releases/download/v4.30.3/yq_linux_amd64 && chmod +x yq_linux_amd64
```

Comment:
yq is already part of ` hack/tools/bin` and there is a make target to install it, it can be installed like other binaries in this file https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2837/files#diff-2009f7be5363812e22862771f557363795759e67b3ca2e32c50d9fd5f8c5669bR31

We don't want to be hardcoding dependency versions in scripts